@here/harp-materials/lib/CirclePointsMaterial.ts

```diff
@@ -117,20 +116,9 @@ export class CirclePointsMaterial extends THREE.ShaderMaterial {
         this.uniforms.size.value = size;
     }
 
-    /**
-     * Get circle opacity.
-     */
-    get opacity(): number {
-        return this.m_opacity;
-    }
-
-    /**
-     * Set circle opacity.
-     */
-    set opacity(opacity: number) {
-        this.m_opacity = opacity;
-
-        // Base constructor may set opacity before uniform being created.
+    setOpacity(opacity: number) {
+        this.opacity = opacity;
+        // Base constructor may set opacity before uniform being created:
```

Comment:
Derive this class from RawShaderMaterial and move this method there.

Reply:
done